### PR TITLE
Feat/add promo banner

### PR DESCRIPTION
**Pull request by abtx**
- TBM 21st November (Monday) with https://github.com/infracost/dashboard/pull/491
 
This PR:

- adds `PromoBanner`
- makes the page title sections uniform
- updates announcement bar text 
- updates gradient on the Download CTA banner

https://loom.com/share/0e18a8a938d447eb96ef9d6e493aabd0

Screenshot 2022-11-17 at 17 53 16
Screenshot 2022-11-17 at 17 53 19

Screenshot 2022-11-18 at 09 22 59

Mobile:

C64642F9-ADE6-4EE5-8028-45428D44AF59

F22CD423-A4C2-4D02-B720-D015940575C8
A0C96F62-AC5F-45FA-8DFE-DD439C569707
6E89BA55-BC2C-4EFC-85E3-72E085747764

Tablet:
IMG_0451

IMG_0450
IMG_0448

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -108,7 +108,8 @@ module.exports = {
     announcementBar: {
       id: 'support_us',
       content:
-        '<span class="announcement-message">If you like Infracost, give it a <a target="_blank" rel="noopener noreferrer" href="https://github.com/infracost/infracost">star on GitHub</a>!<img src="/img/icons/star-white.svg" class="star-right" alt="Star icon" /></span>',
+        // '<span class="announcement-message">If you like Infracost, give it a <a target="_blank" rel="noopener noreferrer" href="https://github.com/infracost/infracost">star on GitHub</a>!<img src="/img/icons/star-white.svg" class="star-right" alt="Star icon" /></span>',
```
Comment (abtx) [inline docusaurus.config.js:111]: Wanted to keep this so we can easily revert, but we can also easily remove [the commit](https://github.com/infracost/docs/pull/329/commits/45e72d94c1c4779cc833c7c63db95c168472de0d).

**Comment by abtx** [comment]
just spotted the issue with announcement bar on mobile 👀

**Comment by abtx** [comment]
just spotted the arrow on mobile as well 👀

File: src/css/base.css [hunk 2]
```diff
@@ -30,6 +30,8 @@
     #6658c5 100%
   );
 
+	--secondary-gradient: linear-gradient(45deg, #151533 0%, #201866 100%);
+
```
Comment (vdmgolub) [inline src/css/base.css:34]: Let's use 2-space tabs here and in other CSS files to be consistent.

**Review by aliscott — APPROVED** [review]

File: src/pages/pricing.js [hunk 3]
```diff
@@ -3,122 +3,195 @@ import PageLayout from '../components/PageLayout';
...
-                  <span className="icon primary"><Check size={18} /></span>
-                  <span>SSO</span>
-                </li>
-              </ul>
-              <div className="action">
-                <a href="https://dashboard.infracost.io" className="button primary">Start free trial</a>
+              <div className="plan__wrapper">
+                <div className="price__wrapper">
+                  <span>
+                    <span className="price__tier">$30</span>
```
Comment (vdmgolub) [inline src/pages/pricing.js:91]: Can we extract the 50 and 30 values into constants like `PRICE_PER_SEAT` and `PROMO_PRICE_PER_SEAT` so we can reuse them and make it easier to search where we use them?

**Pull request merged**